wcivf/apps/core/models.py

```diff
@@ -18,7 +18,7 @@ class LoggedPostcode(TimeStampedModel):
     def __str__(self):
         return "{0}".format(self.postcode)
 
-def log_postcode(log_dict, blocking=True):
+def log_postcode(log_dict, blocking=False):
```

Comment:
Can you explain this commit? Currently this change on its own will cause a write query per request to the 'postcode' page. Maybe the tests need to be updated rather than the code?